src/components/v5/common/ActionSidebar/partials/forms/BatchPaymentForm/BatchPaymentForm.tsx

```diff
@@ -32,27 +29,7 @@ const BatchPaymentForm: FC<ActionFormBaseProps> = () => {
       >
         <TeamsSelect name="from" />
       </ActionFormRow>
-      <ActionFormRow
-        icon="scales"
-        fieldName="decisionMethod"
-        tooltips={{
-          label: {
-            tooltipContent: formatText({
-              id: 'actionSidebar.tooltip.decisionMethod',
-            }),
-          },
-        }}
-        title={formatText({ id: 'actionSidebar.decisionMethod' })}
-      >
-        <FormCardSelect
-          name="decisionMethod"
-          placeholder={formatText({
-            id: 'actionSidebar.decisionMethod.placeholder',
-          })}
-          options={decisionMethods}
-          title={formatText({ id: 'actionSidebar.availableDecisions' })}
-        />
-      </ActionFormRow>
+      <DecisionMethodField />
```

Comment:
Wasn't the decision method component extracted in #1882 ? Or is this just a case of needing a rebase ?

Reply:
That PR is still open, so I guess whoever merges first?